quick-start/QUICK_START.md

```
@@ -0,0 +1,123 @@
+
+### Prerequisites
+
+- **kubectl** configured to access your cluster
+- **Helm** v3.8+ installed
+- **kind** 
+
+#### Install kind
+
+*macOS* 
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)